packages/rum/test/viewTracker.spec.ts

```diff
@@ -214,5 +216,21 @@ describe('rum view measures', () => {
       resourceCount: 0,
       userActionCount: 0,
     })
+    restorePageVisibility()
+  })
+
+  it('should not collect firstContentfulPaint if page is not visible', () => {
+    setPageVisibility('hidden')
+    const lifeCycle = new LifeCycle()
+    setup({ addRumEvent, lifeCycle })
+    lifeCycle.notify(
+      LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED,
+      FAKE_NAVIGATION_ENTRY as PerformanceNavigationTiming
+    )
+    history.pushState({}, '', '/bar')
+
+    expect(getEventCount()).toEqual(3)
+    expect(getViewEvent(1).view.measures.firstContentfulPaint).toBeUndefined()
+    restorePageVisibility()
```

Comment:
is this test still suited to the strategy?
removing visibility hidden would not fail the test

Reply:
No. I'll rework it.